libopenarc/arc.c

```diff
@@ -1634,14 +1634,29 @@ arc_process_set(ARC_MESSAGE *msg, arc_kvsettype_t type, u_char *str,
 	  /* these have no defaults */
 	  case ARC_KVSETTYPE_SEAL:
 		/* make sure required stuff is here */
-		if (arc_param_get(set, (u_char *) "cv") == NULL)
+		if (arc_param_get(set, (u_char *) "cv") == NULL ||
+		    arc_param_get(set, (u_char *) "t") == NULL)
 		{
 			arc_error(msg, "missing parameter(s) in %s data",
 			          settype);
 			set->set_bad = TRUE;
 			return ARC_STAT_SYNTAX;
 		}
 
+
+		/* check set for duplicates */
+		struct arc_plist *param;
```

Comment:
These variable definitions should be at the top of the function rather than floating inside a switch statement.  There's nothing wrong with it in terms of C validity; the only reason for this request is consistency with the rest of the code.